Makefile

```diff
@@ -44,9 +44,7 @@ lint:
 		black $(ALL_PY_SRCS) && \
 		isort $(ALL_PY_SRCS) && \
 		flake8 $(ALL_PY_SRCS) && \
-		mypy $(PY_MODULE) test/ && \
```

Comment:
Can we please just comment this out? I understand we don't want to block this work but I think we should still strive to have the API documented.

Reply:
[b1eb376](https://github.com/trailofbits/sigstore-python/pull/12/commits/b1eb3760859b1d1a8fe4ffb87867eeb0d6734a91)

Reply:
Oh, this was a mistake, I meant to just remove `interrogate` and `git diff`. Thanks for catching!

Reply:
Yep, that's what I mean though. Shouldn't we still try to aim to have a docstring for everything?

I've temporarily commented it out and will make an issue to reenable interrogate and get it passing.